extension/pybindings/README.md

```diff
@@ -2,8 +2,7 @@
 This Python module, named `portable_lib`, provides a set of functions and classes for loading and executing bundled programs. To install it, run the fullowing command:
 
 ```bash
-EXECUTORCH_BUILD_PYBIND=ON \
-pip install . --no-build-isolation
+CMAKE_ARGS="-DEXECUTORCH_BUILD_XNNPACK=ON" pip install . --no-build-isolation
```

Comment:
Do we build xnnpack by default?

Reply:
I'm turning it on by default here https://github.com/pytorch/executorch/pull/9773. I'll update the PR once that lands